src/frontend/læremidler/steg/2-utdanning/Utdanning.tsx

```diff
@@ -113,6 +117,12 @@ const Utdanning = () => {
 
     const oppdaterValgteAktiviteter = (nyeValgteAktiviteter: EnumFlereValgFelt<string>) => {
         settValgteAktiviteter(nyeValgteAktiviteter);
+        if (
+            !harValgtRegisterAktivitetPåVgsNivåEllerAnnet(nyeValgteAktiviteter, registerAktiviteter)
+        ) {
+            setterLærlingEllerLiknende(undefined);
+            settHarTidligereFullførtVgs(undefined);
+        }
```

Comment:
Her burde vi nog nullstille valideringen og
```suggestion
        if (
            !harValgtAktivitetPåVgsNivå(annenUtdanning, nyeValgteAktiviteter, registerAktiviteter)
        ) {
            setterLærlingEllerLiknende(undefined);
            settHarTidligereFullførtVgs(undefined);
            settValideringsfeil((prevState) => ({
                ...prevState,
                erLærlingEllerLiknende: undefined,
                harTidligereFullførtVgs: undefined,
            }));
        }
```

Reply:
Takk 👍